pkgs/by-name/ra/rastertoezpl/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
If possible, package definitions should use `finalAttrs` instead of `rec`, see https://github.com/NixOS/nixpkgs/blob/0aa475546ed21629c4f5bbf90e38c846a99ec9e9/doc/stdenv/stdenv.chapter.md#fixed-point-arguments-of-mkderivation-mkderivation-recursive-attributes .